editorconfig-eclipse-plugin/editorconfig-core-java

```diff
@@ -1 +0,0 @@
-Subproject commit 9ff6e78159540f5c305551237fcf2c7f14701b1d
```

Comment:
The submodule is still missing. Here is the fix:
```
rm .gitmodules
git submodule add git@github.com:editorconfig/editorconfig-core-java.git org.eclipse.editorconfig.core/editorconfig-core-java
cd org.eclipse.editorconfig.core/editorconfig-core-java
git checkout 9ff6e78159540f5c305551237fcf2c7f14701b1d
```

You can pull the fix from my commit: https://github.com/ncjones/editorconfig-eclipse/commit/00f8e99154bf57f6c40003fa21f3ca92cdb9491c. I'll merge once that change is applied.